### PR TITLE
Sprint_19

### PR DESCRIPTION
**Pull request by TOxaREY**

### CODE REVIEW

File: app/src/main/java/xyz/toxarey/playlistmaker/media_library/ui/MediaLibraryFragment.kt [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                0 -> tab.text = getString(R.string.favorite_tracks)
+                1 -> tab.text = getString(R.string.playlists)
+            }
+        }
+
+        tabLayoutMediator.attach()
+    }
+
+    override fun onDestroyView() {
+        super.onDestroyView()
```
Comment (i-masloed) [inline app/src/main/java/xyz/toxarey/playlistmaker/media_library/ui/MediaLibraryFragment.kt:56]: Рекомендация: tabLayoutMediator.detach() лучше вызвать до вызова  super.onDestroyView()

File: app/src/main/java/xyz/toxarey/playlistmaker/media_library/ui/PlaylistsFragment.kt [hunk 2]
```diff
@@ -10,26 +10,20 @@ import xyz.toxarey.playlistmaker.databinding.FragmentPlaylistsBinding
...
-    private val binding get() = _binding!!
+    private lateinit var binding: FragmentPlaylistsBinding
 
     override fun onCreateView(
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View? {
-        _binding = FragmentPlaylistsBinding.inflate(
+        binding = FragmentPlaylistsBinding.inflate(
```
Comment (i-masloed) [inline app/src/main/java/xyz/toxarey/playlistmaker/media_library/ui/PlaylistsFragment.kt:20]: Рекомендация: гугл рекомендует подход с внутренней переменной _binding ( как было до изменений), Предпочтительнее оставить как было
